browser/src/control/Toolbar.js

```diff
@@ -1153,6 +1153,14 @@ L.Map.include({
 			this.sendUnoCommand('.uno:DeletePrintArea');
 			this.print();
 			break;
+		case 'savecomments':
+			if (this._map.isPermissionEditForComments()) {
```

Comment:
did you test this code? does it work?

I think `this` here is `map` object already

Reply:
I put this back in draft, it wasn't tested yet.
This is fixed now.